aws/v9.0.7/release.yaml

```
@@ -0,0 +1,50 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v9.0.7
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - name: chart-operator
+    version: 0.13.0
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.34.0
+    name: nginx-ingress-controller
+    version: 1.7.2
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 5.5.3
+  - name: calico
+    version: 3.9.6
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 0.23.12
+  - name: containerlinux
+    version: 2512.2.0
```

```suggestion
    version: 2512.2.1
```

On popular demand (see https://gigantic.slack.com/archives/CSBNSV58C/p1594735082293100), will give flatcar 2512.2.1 a chance.